app/src/main/res/values/strings.xml

```diff
@@ -86,6 +86,8 @@
     <string name="title_select_date">Select date</string>
     <string name="title_select_start_day">Start date</string>
     <string name="title_select_end_day">End date</string>
+    <string name="title_skip_login">Skip Login</string>
+    <string name="title_logout">Logout</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_logout` appears to be unused
 Unused resources make applications larger and slow down builds.